Sources/PointFree/Admin/Index.swift

```diff
@@ -82,40 +82,49 @@ private func sendNewEpisodeEmails<I>(_ conn: Conn<I, Episode>) -> IO<Conn<I, Pre
 
 private func sendEmail(forNewEpisode episode: Episode, toUsers users: [Database.User]) -> EitherIO<Prelude.Unit, Prelude.Unit> {
 
-  return lift <| IO {
-    let newEpisodeEmails = users.map { user in
-      sendEmail(
-        to: [user.email],
-        subject: "New Point-Free Episode: \(episode.title)",
-        unsubscribeData: (user.id, .newEpisode),
-        content: inj2(newEpisodeEmail.view((episode, user)))
-        )
-        .delay(.milliseconds(100))
-        .retry(maxRetries: 3, backoff: { .seconds(10 * $0) })
+  // A personalized email to send to each user.
+  let newEpisodeEmails = users.map { user in
+    lift(IO { inj2(newEpisodeEmail.view((episode, user))) })
```

Comment:
creating separate IO for rendering the emails cause it takes a bit of time to render 300+ emails